governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+## SDK Community
+
+The community is organized as follows:
+
+- Every SDK has its own
+  [Github Team](https://github.com/orgs/cloudevents/teams), for example
+  `sdk-go-maintainers` is the group of mantainers for
+  [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
+- The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
+
+## New maintainers
+
+TBD
+
+## Ensuring projects health
+
+In order to ensure the health of the SDK projects, we define the following
+actions. These are crucial to show the commitment of the community to develop
+and support these libraries, which are an important part of the CloudEvents
+ecosystem.
+
+We define an SDK project _healthy_ if:
+
+1. It works with the latest version of the programming language
+2. It supports the latest versions of the integrated libraries/frameworks
+3. It receives security patches regularly
+4. It supports the last N-1 major versions of CloudEvents spec, as defined in
+   [Contribution Acceptance](../SDK.md#contribution-acceptance)
+5. Issues and PRs are triaged (labeled, commented, reviewed, etc) regularly
+
+We define a project `cloudevents/sdk-x` _not actively maintained_ if:
+
+6. No commits on master from `sdk-x-maintainers` since 4 months
+7. Issues and/or PRs are not being triaged from `sdk-x-mantainers` since 2
+   months
+8. Security patches are not being **released** from `sdk-x-mantainers` since 1
+   months from CVE disclosures
+
+To prevent projects _not actively maintained_, we define different actions the
+community might take:
+
+- Temporary security patches delivered by an _sdk maintainer_ not part of
+  `sdk-x-maintainers` group
```

Comment:
This sentence is a bit confusing. Two things stand out. First, what is an sdk maintainer? Is this someone who is in the sdk maintainers group but who is not a maintainer on X repository? Second, if so, how does this remedy the situation where a security patch is not released? Do the sdk maintainers (union of all sdk-x-maintainers) each have publish credentials for all SDKs? That seems insecure.

Reply:
I tried to explain it in the first section of the document the difference between _sdk maintainer_ and `sdk-x-maintainer`, how could i reword this sentence?

About having the rights to publish release, we had a discussion a while ago with @n3wscott which proposed to automate it, in order to avoid a single person to hold rights to publish packages. The part of this document about security patches kinda "assumes" that we have such automation in place